### DISCUSSION TITLE
Giscard-AI question generation have many carriage return type character

### DISCUSSION THREAD

**Discussion by subhamku2020** [Giskard-AI/giskard-oss#1958] [Q&A] [answered]
In the below image you can see, I am printing the dataframe for "question being processed" and this have question along with some other character (like Questions being processed: ['How does inflation impact the stock market?\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x0) and because of which the scan is failed  

<img width="1132" alt="image" src="https://github.com/Giskard-AI/giskard/assets/71179793/493114e6-fab9-4d58-85a5-a50c7a0abfa2">

Can you help with this or is this a error from Giskard-AI

**Comment by alexcombessie** [comment]
Hello @subhamku2020 - could you please share with us the full error stack trace in text?

**Comment by kevinmessiaen** [comment, marked as answer]
Hello, sorry for late answer.

It's a special character injection: https://docs.giskard.ai/en/stable/knowledge/llm_vulnerabilities/index.html#example-character-injection-detection

It's expected to fail if you're model is vulnerable to this attack
